src/js/components/AccordionPanel.js

```diff
@@ -0,0 +1,76 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component, PropTypes } from 'react';
+import classnames from 'classnames';
+import Header from './Header';
+import ListItem from './ListItem';
+import TabNextIcon from './icons/base/TabNext';
+import Collapsible from './Collapsible';
+
+import CSSClassnames from '../utils/CSSClassnames';
+
+const CLASS_ROOT = CSSClassnames.ACCORDION_PANEL;
+
+export default class AccordionPanel extends Component {
+  constructor (props) {
+    super(props);
+    this._onClickPanel = this._onClickPanel.bind(this);
+    this.state = {
+      isOpen: props.isOpen || false
+    };
+  }
+
+  componentWillReceiveProps (nextProps) {
+    if (this.props.isOpen !== nextProps.isOpen) {
```

Comment:
`isOpen` isn't in propTypes. I'd suggest `active` as a term, as that would align with other components.